common-ui/src/main/res/layout/component_text_input_view.xml

```diff
@@ -58,8 +58,7 @@
             android:layout_marginTop="@dimen/keyline_3"
             android:hint="Non-editable text with end icon"
             android:text="This is not editable."
-            app:editable="false"
-            app:endIcon="@drawable/ic_copy" />
```

Comment:
why is this removed?

Reply:
Good catch! `endIcon` is set programmatically with the new method. But I will bring back `editable="false"`

Reply:
Actually, `editable="false"` is still there so the change it's just show endIcon programmatically so there is a use case in the preview which is intended.